packages/utils/src/requireEnv.js

```diff
@@ -17,3 +17,11 @@ export const requireEnv = variable => {
   }
   return value;
 };
+
+export const getEnvVarOrDefault = (variable, defaultValue) => {
+  const value = process.env[variable];
+  if (value === undefined) {
+    return defaultValue;
+  }
+  return value;
+};
```

Comment:
Nice, this will come in handy! We've been doing a lot of `process.env[x] || 'default'` about the place which is a tad ugly imo. Will be nice to switch to this 👍